lib/active_merchant/billing/gateways/credorax.rb

```diff
@@ -164,6 +164,7 @@ def authorize(amount, payment_method, options = {})
         add_3ds_2_optional_fields(post, options)
         add_echo(post, options)
         add_submerchant_id(post, options)
+        add_cardholder_name_inquiry(post, options)
```

Comment:
I wonder if this should come after `add_stored_credential` in the order of sub-methods?

The reason being that `a9` is also set in that submethod, either by applying the `transaction_type` that is passed in, or defaulting to `9` if it's a CIT.

In this current order, I think there's a risk that the `a9` value we're setting would get overwritten 😬